nikola/data/samplesite/pages/quickref.rst

```diff
@@ -58,22 +58,22 @@
 
 
         <p>The full details of the markup may be found on the
-        <a href="http://docutils.sourceforge.net/rst.html">reStructuredText</a>
+        <a href="https://docutils.sourceforge.io/rst.html">reStructuredText</a>
```

Comment:
quickref.rst/html is a third-party document. There’s no need to change those links, and if we ever update this file with a new upstream version, those links will go back to http anyway. (You can keep those changes, and  you should make the one change mentioned below, but those changes might be undone in the future if we update it from upstream.)